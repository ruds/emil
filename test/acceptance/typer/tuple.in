();
(2);
(2, "43");
(it, {a = 12, b = it});
(14i : int, "43") : (int * string);
(14, "43") : (int * string);
