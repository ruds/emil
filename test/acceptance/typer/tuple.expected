@0001:
val it : ()
@0002:
val it : bigint
@0003:
val it : (bigint * string)
@0004:
val it : ((bigint * string) * {a: bigint, b: (bigint * string)})
@0005:
val it : (int * string)
@0006: ERROR
@0007:
